modules/ROOT/pages/munit-extensions-maven-plugin.adoc

```diff
@@ -0,0 +1,97 @@
+= Operating the MUnit Extensions Maven Plugin
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Learn how to run your tests using the MUnit Extensions Maven plugin.
```

Comment:
Consider putting statements about MUnit leveraging Java remote debugging and Maven here (if that works.) I think these statements interfere with the main points later on.